locales/locales.php

```diff
@@ -1662,6 +1680,20 @@ public function __construct() {
 		$pap->wp_locale = 'pap';
 		$pap->slug = 'pap';
 
+		$pap_aw = new GP_Locale();
+		$pap_aw->english_name = 'Papiamento (Aruba)';
+		$pap_aw->native_name = 'Papiamento';
+		$pap_aw->lang_code_iso_639_1 = 'pap';
+		$pap_aw->lang_code_iso_639_2 = 'pap';
+		$pap_aw->slug = 'pap-aw';
+
+		$pap_cw = new GP_Locale();
+		$pap_cw->english_name = 'Papiamento (Curaçao and Bonaire)';
```

Comment:
Isn't Curaçao the one we already have in https://github.com/glotpress/GlotPress-WP/blob/9a9bfc2/locales/locales.php#L1656-L1663? See country code. So this locale should probably only for Bonaire which has `BQ` as the country code. Therefore the locale variable is `$pap_bq` with `$pap_bq->country_code = 'bq'`.